### PR TITLE
SRM Elder-class Montagnes' Vessel (Glaive Replacement)

### PR DESCRIPTION
**Pull request by Latentish**
## About The Pull Request
Replaces the Glaive with the Elder-class.
image
image
elder class real

## Why It's Good For The Game
The Glaive is old and a maptainer has requested I maintain my maps and replace or revamp it. I chose the former.
This version gives more attention to the non-glade sections and makes the glade less janky. This version also puts more of a focus on the non-mining and ruin running aspects of the old ship. The ashie med, botany, animals, and ashwines.
You cannot actually fish in the ship. No I will not be making it so you can. Not only would that not make sense, but it would require porting over normal fishability from the planetary turfs onto the shipturf version of water. Which I'm just not doing.

## Changelog

:cl:phoaly
add: Added the Elder-class
del: Removed the Glaive-class
/:cl:

### CODE REVIEW

**Comment by JimKil3** [comment]
This is my first time reviewing a map PR, so don't take my word as gospel, lol.
image
I'm pretty sure having vents and scrubbers directly adjacent isn't best practice. I'll only use this one example, but this pops up in a lot of other places.
image
The terminal here being inaccessible is kind of weird. Maybe move the SMES off to the side, then have the PACMANs side-by-side?
image
Does this room need 2 vents/scrubbers? Also, this is entirely my opinion, but I'm not sure if a personal AI fits the theme of the SRM.
image
Instead of mapping in the contents like this, I'd personally add the items that are in the hunter/shadow lockers to the locker's `PopulateContents` proc - the lockers are already a subtype.
image
Same with the personal AI, I'm not so sure if this laptop fits the SRM theme.
image
These two bottles are an abstract subtype - not meant to be seen by players.
image
I think you want this welder to be an industrial welder, since other ships have an industrial welder on top of the toolboxes like this.

**Comment by JimKil3** [comment]
Also, you could use the wooden computer frames SolGov uses to match the rest of the ship more with the consoles

**Review by meemofcourse — CHANGES_REQUESTED** [review]
imagen
the Goat attacks other mobs in the area
imagen
i feel like it would be better to use a ammo can. also, 50 x 6 is 300 rounds of ammunition. maybe halve this
imagen
i don't think this ship warrants a crew monitoring console, and the modcomp laptop looks out of place
imagen
not sure if the doubled sermon holodisk is intentional
imagen
the shotgun subtype of the guncase generates shotguns. there are also winchesters in a shotgun locker. please use the normal guncase
imagen
imagen
you don't need this much amount of GPS
imagen
do you really need two scrubbers and two vents for this room

**Review by goober3 — APPROVED** [review]
gee golly... what a nice ship. i dare say, this is ready for a test merge!

File: _maps/configs/srm_elder.json [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
+	"map_short_name": "Elder-class",
+	"map_path": "_maps/shuttles/roumain/srm_elder.dmm",
+	"description": "A standard issue vessel to the highest ranks of the Saint-Roumain Militia. While “standard”, this class of vessel is unique to the Montagne that owns it. Each ship is designed around a central garden consisting of plants, soil, and a tree from the owning Montagnes’ home planet. As a highly religious ascetic order, the SRM supplies each Glaive with supplies to farm, raise animals, and perform medicine in more “natural” ways, using herbs and plants grown in house. The ship is captained by a Montagne, who oversees a team of Hunters, and Shadows apprenticing them.",
```
Comment (goober3) [inline _maps/configs/srm_elder.json]:
```suggestion
	"description": "A standard issue vessel to the highest ranks of the Saint-Roumain Militia. While “standard”, this class of vessel is unique to the Montagne that owns it. Each ship is designed around a central garden consisting of plants, soil, and a tree from the owning Montagnes’ home planet. As a highly religious ascetic order, the SRM supplies each Elder-class with supplies to farm, raise animals, and perform medicine in more “natural” ways, using herbs and plants grown in house. The ship is captained by a Montagne, who oversees a team of Hunters, and Shadows apprenticing them.",
```

**Review by goober3 — CHANGES_REQUESTED** [review]
3305554a0809b6d6e9cf054815527c81
these guns should be more secure. put them behind windoors, perhaps.

1de0856181d37f2b26277ebfacb76805
these don't fit on an SRM ship.

4c6672abf2bca2932d9edbcfa5098828
this torch clips with the window above it. pixel shift it down a little.

e386562547ed24e8c9396ffcdedecbf7-1
replace this manifold with the non-visible type. vary up the flooring and the decals in this airlock a bit, we want to avoid mono-tile decal-less rooms. 

9e4d185ed64fe2d7d8e5939865afab89
vary up the tiles in the engine room, too. make it practical, while still fitting the SRM's vibe.

6490fbeba4e33ce913d9913936367710
this cargo bay seems like quite a major stylistic change to the rest of the ship. make this spot resemble the equipment room more, perhaps. 

633e18ff8d953c475d43adb13841aa15
don't use this type. replace these with independent mining hardsuits, if you like. but i'm not 100% on the SRM using hardsuits at all.

8e6ae99ba24ef1605c18ee52483a697a
vary up the flooring in the medbay, please. wooden flooring would suffice. 'sterile' tiles towards the operating area, perhaps. 

all in all, this is a major improvement to the glaive. good job! the style and the decalling just needs clearing up, in places. consider adding more room for cargo, also.

**Comment by Skrem7** [comment]
Stripper clips in the ammo box are .300 BLK, not .300 Magnum. I.E. they don't fit the Smile rifle. I'd make a new stripper clip of the same sprite, but just load it with .300 Magnum to accompany the fact that this rifle's actually in the game now.

**Review by rye-rice — APPROVED** [review]
357 ammo box sprite by me

**Comment by Latentish** [comment]
No mergies until #2618 gets merged (needs the .300 magnum ammo clip)

**Review by Zevotech — CHANGES_REQUESTED** [review]
I'm not 100% on the torches for lighting but it is unique and pretty cool as a gimmick so I'll concede that you don't have to change it.

Plushies layer behind chairs. Also, it feels like there's too many plushies on this ship for an asetic group of religious hunters.
image

Crates don't map onto crate shelves. Also, the fishing toolboxes already come with worms, line, hook and a fishing rod. Just make it have 3 fishing toolboxes and a fishing book or two. Additionally, having a whole crate dedicated to one disk feels wrong, maybe make the other crate hold additional ammo or possibly more disks.
image

A copy of the SRM lore book (maybe include it in their backpacks in the outfit when they spawn?) and a lore holodisk feels excessive, but it's excusable. What isn't excusable is the witchhunter garb. That is chaplain armor and does not at all fit with the ship. They also do not need 3 pairs of boots. Pick a pair that best fits them and go with it. Maybe give black cowboy boots for the hunters and brown ones for the shadows. 
Remove the zippo lighters and lanterns, as those are already in the cryo room. Move the GPS devices from the cryo room into the lockers, one for each person.
image

On the topic of equipment, remove the similarly crusty chaplain riot armor from the montagnes locker. Move the 762x38 ammo boxes to the armory if theres no gun that uses them in the locker. Same policy with the boots/lighters/adding a GPS as the previous comment.
image

Too many seeds. I'd say keep the normal ones and make them forage for the lavaland flora at least.
image

I have no idea why you would do this. Putting two of the same vent/scrubber next to eachother makes no sense. Either make it vent/scrubber and vent/scrubber or just make it one of each. Same for the bridge.
image

These tiles are always lit. Change them to something else or fix the lighting issue.
image


Sigh. Not fond of the tone this uses for a semi-religious tutorial book and also I'm fairly certain "clowns" would be extremely offensive to call readers in-lore. I understand you didn't write this so its fine but if possible I'd take out clowns/some of the more egregious wordings and even slate it for a full rewrite by the lore team if you can.
image
image

**Review by Zevotech — CHANGES_REQUESTED** [review]
Remove the witchhunter's stuff from the Hunter's lockers. It's crufty, ugly and is still literal riot armor. This is the single hill in this review I will die on.
Also remove the excess fishing equipment. The fishing toolboxes come pre-stocked. All you need are the toolboxes and fishing manuals.
image

Fix the scrubber/vent distribution in the bridge. Two of each next to eachother looks weird. Make it like how the starboard hall ones are.
image

HP guns in the lockers have the default sprites ingame. Seems to be a bug. Ask rye.
image

These doors have no access restrictions. Unless this is intentional, make it so that the Hunters are the ones that can access the guns.
image

Exterior hull here still uses the lit external areas, change it to the dark kind.
image

Just a mild gripe, these buttons don't look very good on wooden tables. I'd move them to a wall, but I wouldn't care if it was left as-is.
image

This medkit is empty. If you want them to have a filled one, use the /standard type of it.
image

**Comment by Latentish** [comment]
also removed witchhunter gear and the cultpack, and changed the external area to /dark

**Review by meemofcourse — APPROVED** [review]
me when i take 17 trickwine and start seeing the bull

**Review by Zevotech — APPROVED** [review]

**Review by goober3 — APPROVED** [review]

**Pull request merged**